plugins/modules/zos_operator.py

```diff
@@ -286,6 +289,23 @@ def run_operator_command(params):
     wait_s = params.get("wait_time_s")
     cmdtxt = params.get("cmd")
 
+    zv = ZOAU_API_VERSION.split(".")
```

Comment:
Can we instead use module_utils/zoau_version_checker instead? If any changes occur in the future then we would just need to change in one place.

Reply:
I'd like to create a new ticket for that change, based on Wednesday's conversation.  It works, but I'll make the change so that zoau_version_checker calls the new code, and then this can use zoau_version_checker.

Reply:
Please share the issue here so we can resolve this conversation, thanks @richp405

Reply:
#1018